core/domain/subtopic_page_services_test.py

```diff
@@ -23,10 +23,22 @@
 from core.platform import models
 from core.tests import test_utils
 
+import feconf
+
 (base_models, topic_models, ) = models.Registry.import_models([
     models.NAMES.base_model, models.NAMES.topic])
 
 
+# TODO: Remove this mock class and the migrate_page_contents tests
```

Comment:
Please file an issue and reference it here so this does not get lost.

Reply:
How about I file an issue after this PR gets merged? That way I could link this TODO in that issue. Does that sound good?

Reply:
I think it makes more sense to reference the issue in-code, not sure if what the standard is though. @seanlip, WDYT?

Reply:
On other files, I just added this TODO and made no issue link

Reply:
Firstly, all TODOs should be formatted `TODO(lilithxxx):`. The name in the parentheses is the person to ask for more context on the issue. If you have added "naked TODOs" in the past please make sure to fix those (you can do it in this PR).
/cc @apb7 @oppia/dev-workflow-team this is an important dev workflow check that we should implement.

Also, please file the issue now and add it here. You can update the text of that issue later. Please do this as a general practice; filing the issue is sometimes part of resolving the review comments.

Reply:
Done and opened issues #7010 and #7009